### PR TITLE
Fix twig

### LINKED ISSUE #7668: issue with 4.6.0
Just upgraded to 4.6.0 on my project and noticed test fails

```
00:07:12.255 Failed asserting that the HTTP status code is 200! Actual status code is 500
00:07:12.255 Unexpected operator of value "=". Did you try to use "===" or "!==" for strict comparison? Use "is same as(value)" instead.
00:07:12.255 Twig\Error\SyntaxError: Unexpected operator of value "=". Did you try to use "===" or "!==" for strict comparison? Use "is same as(value)" instead. in /var/www/.../symfony/vendor/sonata-project/admin-bundle/src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig:77
``` 

I'm using twig `v3.3.4`

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

I am targeting this branch, because {bugfix}.

Closes #7668.

## Changelog

```markdown
### Fixed
- Model_autocomplete_type template
```

### CODE REVIEW

File: src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig
Comment (dmaicher) [inline src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig]: I guess `!==` also does not work? let me try

Comment (VincentLanglet) [reply src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig]: indeed

Comment (VincentLanglet) [reply src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig]: fixed

**Review by core23 — APPROVED** [review]

**Comment by VincentLanglet** [comment]
I wait for your approval to merge/release this @dmaicher

**Review by dmaicher — APPROVED** [review]
yes this seems to work fine.

**Comment by dmaicher** [comment]
I think we should lint twig templates as part of CI?

```
bin/console lint:twig vendor/sonata-project/admin-bundle/src/Resources/

  ERROR  in vendor/sonata-project/admin-bundle/src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig (line 77)
      75                         url:  '{{ url ?: path(route.name, route.parameters|default([])) }}',
      76                         dataType: 'json',
  >>  77                         delay: {{ delay === 100 and quiet_millis !== 100 ? quiet_millis : delay }}, // NEXT_MAJOR: Replace by `{{ delay }}` instead.
  >> Unexpected operator of value "=". Did you try to use "===" or "!==" for strict comparison? Use "is same as(value)" instead. 
      78                         cache: {{ cache ? 'true' : 'false' }},
      79                         processResults: function (data, params) {

                                                                                                                        
 [WARNING] 127 Twig files have valid syntax and 1 contain errors.

**Comment by VincentLanglet** [comment]
> I think we should lint twig templates as part of CI?
> 
> ```
> bin/console lint:twig vendor/sonata-project/admin-bundle/src/Resources/
> 
>   ERROR  in vendor/sonata-project/admin-bundle/src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig (line 77)
>       75                         url:  '{{ url ?: path(route.name, route.parameters|default([])) }}',
>       76                         dataType: 'json',
>   >>  77                         delay: {{ delay === 100 and quiet_millis !== 100 ? quiet_millis : delay }}, // NEXT_MAJOR: Replace by `{{ delay }}` instead.
>   >> Unexpected operator of value "=". Did you try to use "===" or "!==" for strict comparison? Use "is same as(value)" instead. 
>       78                         cache: {{ cache ? 'true' : 'false' }},
>       79                         processResults: function (data, params) {
> 
>                                                                                                                         
>  [WARNING] 127 Twig files have valid syntax and 1 contain errors.                                                       
>                                                                                                                         
> ```

Can you open an issue on dev-kit ?

**Pull request merged**
